deploy/cephcsi/image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+
+FROM centos:7
+LABEL maintainers="Kubernetes Authors"
+LABEL description="Ceph-CSI Plugin"
+
+ENV CEPH_VERSION "mimic"
+
+RUN yum install -y centos-release-ceph && \
+    yum install -y ceph-common kmod ceph-fuse attr e2fsprogs xfsprogs rbd-nbd && \
+    yum clean all
+
+COPY cephcsi /cephcsi
```

Comment:
Can we use a more standard location instead of `/`? Perhaps `/opt/ceph-csi/`. 

This is what LFH says about it: 
`
 This directory is reserved for all the software and add-on packages that are not part of the default installation. `.

Reply:
I'd prefer that too. I almost changed it to /usr/local/bin/cephcsi but wanted to keep the delta small at first. /opt would also be ok. Next time I push I will change the path unless someone else has strong feelings about leaving the binary in /.